src/Tribe/Utils/Coordinates_Provider.php

```diff
@@ -31,7 +31,7 @@ class Tribe__Utils__Coordinates_Provider {
 	/**
 	 * @var WP_Http
 	 */
-	private $http;
+	private $https;
```

Comment:
This is really just a small quibble, but I'd probably have left the property name as `http` despite the change in default protocol to `https://` on line 14, just because that's consistent with the object type (`WP_Http`).

It's also slightly inconsistent with the name of existing method `get_http()` (which we can't really change without deprecating and replacing it).